.github/ISSUE_TEMPLATE/bug_report.md

```
@@ -0,0 +1,29 @@
+---
+name: Bug Report
+about: Create a report to help us improve
+title: "fix: "
+labels: bug
+---
+
+**Description**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)